myokit/formats/mathml/_parser.py

```diff
@@ -100,6 +100,9 @@ class MathMLParser(object):
         Becomes a :class:`myokit.Number`. To process units which may be present
         in the tag's attributes (esp. in CellML) the number post-processing
         function can be used.
+    ``<csymbol>``
```

Comment:
Can you explain in this bit that the name generator is used? (The docstring below won't be visible to user)